### PR TITLE
Made a landing page for arch 

### LINKED ISSUE #2: Change design
We need to add an Arch Linux design for the landing page. The design files should be placed in: `static/themes/archlinux`. Check out `static/themes/fedora` as example.

### PR DESCRIPTION
**Pull request by sonuishaq67**
#7 
> Closes #2
>  - [x] Made a sample questions page (i.e) arch.yml
>  - [x] Updated build.sh
>  - [x] Updated Readme
>  - [x] added icons and logo
>  - [x] Also updated the fork to latest version of fedora-infra/asknot-ng

If there's anything to be changed please do inform and ill change it

### CODE REVIEW

File: questions/arch.yml [hunk 1]
```diff
@@ -0,0 +1,128 @@
+title: What can I do for Arch?
```
Comment (polyzen) [inline questions/arch.yml:1]: [questions/archlinux.yml](https://github.com/archlinux/asknot-ng/blob/archlinux/questions/archlinux.yml) already exists, so you can delete this.

Comment (sonuishaq67) [reply questions/arch.yml:1]: Okays

File: build.sh [hunk 2]
```diff
@@ -3,8 +3,7 @@
...
 python3 asknot-ng.py \
-	./templates/index.html \
-	./questions/fedora.yml \
-	./l10n/fedora/locale \
-	-s ./static \
-	-t fedora
+	templates/index.html \
+	questions/arch.yml \
+	l10n/arch/locale \
+	--theme arch
```
Comment (polyzen) [inline build.sh:9]: archlinux*

File: README.md [hunk 3]
```diff
@@ -57,8 +57,10 @@ Create a virtualenv into which you can install the module.
 
 Run the script with the Fedora configuration::
 
-    $ ./asknot-ng.py templates/index.html questions/fedora.yml l10n/fedora/locale --theme fedora
-    Wrote build/en/index.html
+    $ ./asknot-ng.py templates/index.html questions/arch.yml l10n/arch/locale --theme arch
```
Comment (polyzen) [inline README.md:60]: archlinux*

File: README.md [hunk 4]
```diff
@@ -33,20 +33,20 @@ We have a [Fedora instance up and running][wcidff] if you’d like to poke it.
 The site-generator script is written in Python, so you’ll need that.
 Furthermore, see [requirements.txt][requirements] or just run::
 
-    $ sudo dnf install python-mako PyYAML python-virtualenv
+    $ sudo pacman -S python-yaml python-mako python-virtualenv
```
Comment (polyzen) [inline README.md:36]: May as well keep `python-mako` before `python-yaml` to keep it alphabetical.

File: README.md [hunk 5]
```diff
@@ -57,8 +57,10 @@ Create a virtualenv into which you can install the module.
 
 Run the script with the Fedora configuration::
 
-    $ ./asknot-ng.py templates/index.html questions/fedora.yml l10n/fedora/locale --theme fedora
-    Wrote build/en/index.html
+    $ ./asknot-ng.py templates/index.html questions/arch.yml l10n/arch/locale --theme arch
+    
+    
+Wrote build/en/index.html
```
Comment (polyzen) [inline README.md:63]: Should keep this line indented with the command as this is part of the command's output.

**Review by sonuishaq67 — COMMENTED** [review]
> archlinux*

this is only for questions right 
not the locale or theme

Edit : sorry i didnt mean to review

File: build.sh (hunk 2 above)
Comment (polyzen) [inline build.sh]: archlinux*

Comment (sonuishaq67) [reply build.sh]: yeah i changed it from arch.yml to archlinux.yml thats it right?
or Should i change the directory names too?

**Comment by polyzen** [comment]
> > archlinux*
> 
> this is only for questions right
> not the locale or theme

Should probably stick to either `arch` or `archlinux` for the questions, theme, and locale. @shibumi, any opinions?

**Comment by sonuishaq67** [comment]
In my opinion arch looks better but there's a shell command with that name (arch to Rebuild a list's archive.) or for better readablity archlinux is better as people don't think arch means architecture

**Comment by polyzen** [comment]
`sonuishaq67 wants to merge 27 commits into archlinux:archlinux from sonuishaq67:develop`
The commits should also be on the `archlinux` branch of your repo. This should read `into archlinux:archlinux from sonuishaq67:archlinux`.

Do your changes rely on the upstream commits that haven't been merged yet? It would be simpler to review if just the landing page commits were in their own merge request.

**Comment by sonuishaq67** [comment]
Okay so Ill merge my develop with archlinux and close this pr and send one again ? @polyzen 
> Do your changes rely on the upstream commits that haven't been merged yet? 

No I fetched from the develop branch from upstream which was the default branch 

>  Edit: In the meantime: 

Anything wrong with the pr ??

**Comment by polyzen** [comment]
You can use this branch for a pull request: https://github.com/polyzen/asknot-ng/tree/landing-page
The landing page commits were cherry-picked to the `archlinux` branch and the history was cleaned up.

I've split off the merge of the latest upstream commits to https://github.com/archlinux/asknot-ng/pull/9

**Comment by sonuishaq67** [comment]
Okay sure

**Pull request closed without merging**